frontend/map.service.ts

```diff
@@ -4,19 +4,27 @@ import { Http, Response } from "@angular/http";
 import { MapObject } from "./mapObject";
 import { Observable } from "rxjs/Observable";
 
+export enum MapObjectType {
```

Comment:
wäre das nicht besser nebem der Klasse `MapObject` in `./mapobject.ts` aufgehoben?